### PR TITLE
provide install script for the latest starcoin and mpm

### PR DESCRIPTION
**Pull request by yuliyu123**
provide install script for the latest starcoin and mpm from the lastest  GitHub release page.

### CODE REVIEW

**Comment by yuliyu123** [comment]
https://github.com/starcoinorg/starcoin/issues/3550

File: scripts/install_starcoin_mpm.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Provide a script to download the latest starcoin and mpm from gitHub release page.
+
+if [ -z "$1" ]; then
+    echo "Usage: $0 <version>"
+    exit 1
+fi
+
+INSTALL_DIR="/usr/bin"
```
Comment (jolestar) [inline scripts/install_starcoin_mpm.sh:10]: 是不是 `/usr/local/bin` 更合适些？

**Review by jolestar — COMMENTED** [review]
能否在 readme 页面提供一个一行安装的命令，类似于

```
curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
```

File: scripts/install_starcoin_mpm.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+else
+  echo "Unsupported system"
+  exit 1
+fi
+
+PACKAGE_DIR="starcoin-${PLATFORM}-latest"
+unzip_dir="starcoin-artifacts"
+PACKAGE="https://github.com/starcoinorg/starcoin/releases/download/${VERSION}/${PACKAGE_DIR}.zip"
+curl -sL -O "${PACKAGE}" && unzip "${PACKAGE_DIR}"
+mv "${unzip_dir}/mpm" "${INSTALL_DIR}" && mv "${unzip_dir}/starcoin" "${INSTALL_DIR}"
```
Comment (yourmoonlight) [inline scripts/install_starcoin_mpm.sh:27]: 这里要不要加个-f ?

Comment (yuliyu123) [reply scripts/install_starcoin_mpm.sh]: Right, fixed.

File: scripts/install_starcoin_mpm.sh (hunk 1 above)
Comment (yuliyu123) [reply scripts/install_starcoin_mpm.sh]: Yeah, fixed.

**Comment by yuliyu123** [comment]
> 能否在 readme 页面提供一个一行安装的命令，类似于
> 
> ```
> curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
> ```

done.

**Review by jolestar — APPROVED** [review]

**Pull request merged**